nimare/meta/ibma/ibma.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+Image-based meta-analysis estimators
+"""
+from ..base import MetaEstimator
+
+
+class MFX_GLM(MetaEstimator):
+    """
+    The gold standard image-based meta-analytic test. Uses contrast and standard
+    error images.
+    """
+    def __init__(self, dataset, n_iters=10000, voxel_thresh=0.001, clust_thresh=0.05,
```

Comment:
I think we'll almost certainly need another layer on top of `dataset` that represents the actual component of a database/dataset that can be submitted for analysis. So I still lean strongly towards making `dataset` an argument of `fit` and not `init`. Otherwise we'll end up in a weird situation where both the `Dataset` and the `MetaEstimator` classes are implementing methods that filter/mutate the data.